netsim/defaults/attributes.yml

```diff
@@ -94,23 +94,23 @@ node:
   group: list
   role: id
   mgmt:
-    ipv4: { type: ipv4, use: id }
-    ipv6: { type: ipv6, use: id }
+    ipv4: { type: ipv4, use: address }
```

Comment:
This is 'address' because the prefix is implicitly assumed to be a /24?

Reply:
No, because specifying subnet mask on an individual management IP address makes no sense. Either it's within the management network (we know how well that works), or it's an external IP address where the mask doesn't matter.